packages/camera/camera/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 0.9.8+1
+
+* Ignore deprecation warnings for upcoming styleFrom button API changes.
```

Comment:
This should be "Ignores" everywhere. See https://github.com/flutter/flutter/wiki/Contributing-to-Plugins-and-Packages#changelog-style

Reply:
Fixed.